app/models/question.rb

```diff
@@ -1,4 +1,5 @@
 class Question < ApplicationRecord
   has_many :answers
+  belongs_to :user
   validates :description, length: { maximum: 400 }, presence: true
```

Comment:
Could we add a `uniqueness` check as well to avoid duplicate questions and answers?